script/Ciel/Day 13/13_01_02_CIEL13_7C.txt

```diff
@@ -261,12 +261,12 @@ But there are also things that won't disappear. The desire to save Tohno Shiki,
 [sha:26fe74883a627a8bfb2dd21f43ae07ebc985e174]{
 -- Page 298, Offset 36148.
 -- 「私は、それを認めます。これから私が侵す命、私が残す憎しみをすべて自分の罪だと認めます。
-"I accept that. I accept that all the lives I'll hurt from now, and all of the hatred that remains, will become part of my sins.
+"I'll accept it. I'll accept that all of the lives I'll harm from now on, and all of the hatred that remains will become part of my sins.
```

Comment:
I'll accept it all. All of the lives I'll harm, and all of the hatred that remains will become part of my sins.